mobgap/gsd/evaluation.py

```diff
@@ -142,6 +147,11 @@ def calculate_unmatched_gsd_performance_metrics(
        Should have the same format as `gsd_list_detected`.
     sampling_rate_hz
         Sampling frequency of the recording in Hz.
+    zero_division_hint : "warn", "raise" or False, default="warn"
+        Controls the behavior when there is a zero division. If set to "warn",
+        affected metrics are set to NaN and a warning is raised.
+        If set to "raise", a ZeroDivisionError is raised.
+        If set to False, the warning is suppressed and the affected metrics are set to NaN.
```

Comment:
Can we have the option to use "NaN" instead of False (so to make it consistent with the other option)

Reply:
And can you specify in what scope that will be applied. Very specifically we safeguard against the reference_gs_duration_s == 0

Reply:
Sorry I wasn't clear on that one. Can we actually use the value np.nan? In the other case, when we put in a 1 th output will be 1. So here it seems natural if we put np.nan, we get np.nan. Or is that a bad idea?